Comparing source compatibility of /home/someone/git/com.github/io7m/jregions/com.io7m.jregions.core/target/com.io7m.jregions.core-0.0.6.jar against /home/someone/.m2/repository/com/io7m/jregions/com.io7m.jregions.core/0.0.4/com.io7m.jregions.core-0.0.4.jar
WARNING: You are using the option '--ignore-missing-classes', i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jregions.core.unparameterized.sizes.AreaSizesBD  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) com.io7m.jregions.core.unparameterized.areas.AreaBD area(com.io7m.jregions.core.unparameterized.sizes.AreaSizeBD)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jregions.core.unparameterized.sizes.AreaSizesBI  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) com.io7m.jregions.core.unparameterized.areas.AreaBI area(com.io7m.jregions.core.unparameterized.sizes.AreaSizeBI)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jregions.core.unparameterized.sizes.AreaSizesD  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) com.io7m.jregions.core.unparameterized.areas.AreaD area(com.io7m.jregions.core.unparameterized.sizes.AreaSizeD)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jregions.core.unparameterized.sizes.AreaSizesF  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) com.io7m.jregions.core.unparameterized.areas.AreaF area(com.io7m.jregions.core.unparameterized.sizes.AreaSizeF)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jregions.core.unparameterized.sizes.AreaSizesI  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) com.io7m.jregions.core.unparameterized.areas.AreaI area(com.io7m.jregions.core.unparameterized.sizes.AreaSizeI)
***  MODIFIED CLASS: PUBLIC FINAL com.io7m.jregions.core.unparameterized.sizes.AreaSizesL  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) com.io7m.jregions.core.unparameterized.areas.AreaL area(com.io7m.jregions.core.unparameterized.sizes.AreaSizeL)
